### PR TITLE
Add some module descriptions and fill package.yml details.

### PR DESCRIPTION
**Pull request by prushin**

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
 license:             BSD3
-author:              "Author name here"
-maintainer:          "example@example.com"
-copyright:           "2020 Author name here"
+author:              "Mikhail Prushinskiy <mprushinsky@gmail.com>"
+maintainer:          "Mikhail Prushinskiy <mprushinsky@gmail.com>, Ilja Tkachuk <tkachuk.labs@gmail.com>, Artem Markov <drownbes@gmail.com>"
+copyright:           "2020 Coingaming <hello@coingaming.io>"
+synopsis:            Lightning Network Daemon (LND) client library for Haskell
+category:            Lightning, Bitcoin, Finance, Network
+description:         You can find documentation at <https://hackage.haskell.org/package/signable/docs/Data-Signable.html hackage>
```
Comment (tim2CF) [inline package.yaml:10]: Fix copy-paste from Signable package

File: package.yaml [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
+github:              "coingaming/lnd-client"
 license:             BSD3
-author:              "Author name here"
-maintainer:          "example@example.com"
-copyright:           "2020 Author name here"
+author:              "Mikhail Prushinskiy <mprushinsky@gmail.com>"
+maintainer:          "Mikhail Prushinskiy <mprushinsky@gmail.com>, Ilja Tkachuk <tkachuk.labs@gmail.com>, Artem Markov <drownbes@gmail.com>"
+copyright:           "2020 Coingaming <hello@coingaming.io>"
+synopsis:            Lightning Network Daemon (LND) client library for Haskell
+category:            Lightning, Bitcoin, Finance, Network
```
Comment (tim2CF) [inline package.yaml:9]: Add Payments

**Review by tim2CF — CHANGES_REQUESTED** [review]

**Comment by tim2CF** [comment]
Also update ChangeLog.md similar to https://github.com/coingaming/signable/blob/master/haskell/ChangeLog.md#version-01
And update README.md

File: package.yaml (hunk 1 above)
Comment (prushin) [reply package.yaml]: thanks, done

File: package.yaml (hunk 2 above)
Comment (prushin) [reply package.yaml]: thanks, done

**Review by tim2CF — APPROVED** [review]

**Pull request merged**